diff -ruN lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/README.multi_encoder lives-0.9.5-pre1/lives-plugins/marcos-encoders/README.multi_encoder
--- lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/README.multi_encoder	2005-03-07 23:50:18.166091696 +0100
+++ lives-0.9.5-pre1/lives-plugins/marcos-encoders/README.multi_encoder	2005-03-08 01:23:38.956020896 +0100
@@ -46,7 +46,7 @@
 libs are missing (e.g. xvidcore). If the above fails because
 Python is not found change the first line of the encoder scripts
 
-#!/usr/bin/python2.3
+#!/usr/bin/python
 
 to point to the correct executable.
 
diff -ruN lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/dirac_encoder.py lives-0.9.5-pre1/lives-plugins/marcos-encoders/dirac_encoder.py
--- lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/dirac_encoder.py	2005-03-07 23:50:18.166091696 +0100
+++ lives-0.9.5-pre1/lives-plugins/marcos-encoders/dirac_encoder.py	2005-03-08 01:23:38.964019680 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 
 """
 dirac_encoder.py
diff -ruN lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/gif_encoder.py lives-0.9.5-pre1/lives-plugins/marcos-encoders/gif_encoder.py
--- lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/gif_encoder.py	2005-03-07 23:50:18.162092304 +0100
+++ lives-0.9.5-pre1/lives-plugins/marcos-encoders/gif_encoder.py	2005-03-08 01:23:38.972018464 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 
 """
 gif_encoder.py
diff -ruN lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/mkv_encoder.py lives-0.9.5-pre1/lives-plugins/marcos-encoders/mkv_encoder.py
--- lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/mkv_encoder.py	2005-03-07 23:50:18.163092152 +0100
+++ lives-0.9.5-pre1/lives-plugins/marcos-encoders/mkv_encoder.py	2005-03-08 01:23:38.980017248 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 
 """
 mkv_encoder.py
diff -ruN lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/mng_encoder.py lives-0.9.5-pre1/lives-plugins/marcos-encoders/mng_encoder.py
--- lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/mng_encoder.py	2005-03-07 23:50:18.162092304 +0100
+++ lives-0.9.5-pre1/lives-plugins/marcos-encoders/mng_encoder.py	2005-03-08 01:23:38.988016032 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 
 """
 mng_encoder.py
diff -ruN lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/mpeg_encoder.py lives-0.9.5-pre1/lives-plugins/marcos-encoders/mpeg_encoder.py
--- lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/mpeg_encoder.py	2005-03-07 23:50:18.165091848 +0100
+++ lives-0.9.5-pre1/lives-plugins/marcos-encoders/mpeg_encoder.py	2005-03-08 01:23:38.995014968 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 
 """
 mpeg_encoder.py
diff -ruN lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/ogm_encoder.py lives-0.9.5-pre1/lives-plugins/marcos-encoders/ogm_encoder.py
--- lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/ogm_encoder.py	2005-03-07 23:50:18.163092152 +0100
+++ lives-0.9.5-pre1/lives-plugins/marcos-encoders/ogm_encoder.py	2005-03-08 01:23:39.003013752 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 
 """
 ogm_encoder.py
diff -ruN lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/theora_encoder.py lives-0.9.5-pre1/lives-plugins/marcos-encoders/theora_encoder.py
--- lives-0.9.5-pre1.org/lives-plugins/marcos-encoders/theora_encoder.py	2005-03-07 23:50:18.165091848 +0100
+++ lives-0.9.5-pre1/lives-plugins/marcos-encoders/theora_encoder.py	2005-03-08 01:23:39.011012536 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 
 """
 theora_encoder.py
diff -ruN lives-0.9.5-pre1.org/lives-plugins/plugins/encoders/multi_encoder lives-0.9.5-pre1/lives-plugins/plugins/encoders/multi_encoder
--- lives-0.9.5-pre1.org/lives-plugins/plugins/encoders/multi_encoder	2005-03-07 23:50:18.209085160 +0100
+++ lives-0.9.5-pre1/lives-plugins/plugins/encoders/multi_encoder	2005-03-07 23:55:11.489499760 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 
 """
 multi_encoder
