--- lives-0.9.8.6/lives-plugins/weed-plugins/Makefile.am.orig	2007-11-26 18:46:29.268731554 +0100
+++ lives-0.9.8.6/lives-plugins/weed-plugins/Makefile.am	2007-11-26 18:46:41.766824891 +0100
@@ -4,7 +4,7 @@
 
 SUBDIRS = gdk
 
-fxdir = $(datadir)/$(PACKAGE)/plugins/effects/realtime/weed
+fxdir = $(libdir)/$(PACKAGE)/plugins/effects/realtime/weed
 
 if IS_DARWIN
 shared_ldflags = -dynamiclib
--- lives-0.9.8.6/lives-plugins/weed-plugins/gdk/Makefile.am.orig	2007-11-26 19:33:43.336376765 +0100
+++ lives-0.9.8.6/lives-plugins/weed-plugins/gdk/Makefile.am	2007-11-26 19:33:57.834165036 +0100
@@ -2,7 +2,7 @@
 
 %.wo: %.c
 
-fxdir = $(datadir)/$(PACKAGE)/plugins/effects/realtime/weed
+fxdir = $(libdir)/$(PACKAGE)/plugins/effects/realtime/weed
 
 if IS_DARWIN
 shared_ldflags = -dynamiclib
--- lives-0.9.8.6/lives-plugins/plugins/playback/video/Makefile.am.orig	2007-11-26 20:03:46.061359765 +0100
+++ lives-0.9.8.6/lives-plugins/plugins/playback/video/Makefile.am	2007-11-26 20:03:59.059376838 +0100
@@ -1,6 +1,6 @@
 ## Process this with automake to create Makefile.in
 
-videopluginsdir = $(datadir)/$(PACKAGE)/plugins/playback/video
+videopluginsdir = $(libdir)/$(PACKAGE)/plugins/playback/video
 
 if IS_DARWIN
 shared_ldflags = -dynamiclib -fno-common
