--- lives-0.9.5-pre2.2/Makefile.am.orig	2005-03-26 21:40:42.000000000 +0100
+++ lives-0.9.5-pre2.2/Makefile.am	2005-04-04 23:09:54.000000000 +0200
@@ -708,7 +708,7 @@
 
 install-exec-local:
 	rm -rf $(DESTDIR)$(bindir)/lives
-	$(LN_S) $(DESTDIR)$(bindir)/lives-exe $(DESTDIR)$(bindir)/lives
+	$(LN_S) $(bindir)/lives-exe $(DESTDIR)$(bindir)/lives
 	cp -f $(srcdir)/smogrify $(DESTDIR)$(bindir) && chmod +rx $(DESTDIR)$(bindir)/smogrify
 	cp -f $(srcdir)/midistart $(DESTDIR)$(bindir) && chmod +rx $(DESTDIR)$(bindir)/midistart
 	cp -f $(srcdir)/midistop $(DESTDIR)$(bindir) && chmod +rx $(DESTDIR)$(bindir)/midistop
